docs/api/python/contrib/onnx.md

```diff
@@ -0,0 +1,49 @@
+# ONNX-MXNet API
+
+## Overview
+
+[ONNX](https://onnx.ai/) is an open format to represent deep learning models. With ONNX as an intermediate representation, it is easier to move models between state-of-the-art tools and frameworks for training and inference..
+
+The `mxnet.contrib.onnx` package refers to the APIs and interfaces that implements ONNX model format support for Apache MXNet.
+
+With ONNX format support for MXNet, developers can build and train models with PyTorch, CNTK, or Caffe2, and import these models into MXNet to run them for inference and training using MXNet’s highly optimized engine.
```

Comment:
Simplify.
...train models with a variety of deep learning frameworks...
Then list the currently supported frameworks with bullets. (easier to maintain too, when we add more later)

Reply:
How about this -
... train models with a [variety of deep learning frameworks](http://onnx.ai/supported-tools)....
I have linked it to the supported tools page on the ONNX website. That way our page never gets outdated.